SDWebImage/SDWebImageDownloader.h

```diff
@@ -188,4 +188,9 @@ typedef NSDictionary *(^SDWebImageDownloaderHeadersFilterBlock)(NSURL *url, NSDi
  */
 - (void)setSuspended:(BOOL)suspended;
 
+/**
+ * Cancel all operations
+ */
+- (void)CancelAllOperations;
```

Comment:
Please align the function name with coding standards so that it starts with a lowercase letter:

`- (void)cancelAllOperations;`

Also might be nice to expand on the comment too rather than it simply repeating the name of the method? For example:
> Cancels all operations in the download queue